Mathlib/Algebra/Ring/Equiv.lean

```diff
@@ -0,0 +1,870 @@
+/-
+Copyright (c) 2018 Johannes Hölzl. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johannes Hölzl, Callum Sutton, Yury Kudryashov
+Ported by: Anatole Dedecker
+
+! This file was ported from Lean 3 source module algebra.ring.equiv
+! leanprover-community/mathlib commit a59dad53320b73ef180174aae867addd707ef00e
+! Please do not edit these lines, except to modify the commit id
+! if you have ported upstream changes.
+-/
+import Mathlib.Algebra.Group.Opposite
+import Mathlib.Algebra.Hom.Ring
+import Mathlib.Logic.Equiv.Set
+
+/-!
+# (Semi)ring equivs
+
+In this file we define extension of `equiv` called `ring_equiv`, which is a datatype representing an
```

Comment:
```suggestion
In this file we define extension of `equiv` called `RingEquiv`, which is a datatype representing an
```
Please check all other comments and docstrings for similar changes.

Reply:
Sorry, that should be fixed now